services/dynamic-scheduler/src/simcore_service_dynamic_scheduler/api/rest/_ops.py

```diff
@@ -0,0 +1,24 @@
+from typing import Annotated
+
+from fastapi import APIRouter, Depends, FastAPI
+from models_library.api_schemas_directorv2.dynamic_services import (
+    DynamicServiceGet,
+)
+
+from ...services import scheduler_interface
+from ._dependencies import (
+    get_app,
+)
+
+router = APIRouter()
+
+
+@router.get("/ops/running-services")
+async def running_services(
+    app: Annotated[FastAPI, Depends(get_app)],
+) -> list[DynamicServiceGet]:
```

Comment:
I highly recommend to implement pagination when you expose listing of unbound collections (i.e. those that can grow indefinitely). We have hit quite some serious troubles with those api

There are in place some libraries to setup pagination very easily. Search for `fastapi_pagination` in the repository and you will find some examples in api_server, datcore-adapter and storage

Reply:
I really do wonder what is the advantage of having this call paginated when the one used by frontend is not.

Reply:
Also, currently paginating this calls, does not make much sense, since the implementation in the background does not support pagination. In the upcoming future we can do it, but not for now